api-report/container-definitions.api.md

```diff
@@ -165,8 +165,6 @@ export interface IContainerContext extends IDisposable {
     readonly connected: boolean;
     // (undocumented)
     readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
-    // (undocumented)
-    readonly existing: boolean | undefined;
```

Comment:
Looks like you need to build locally - That will revert this, and the type validation changes for IContainerContext.  And first you'll want to address Navin's comment about updating IRuntimeFactory itself (if I'm not mistaken)